test/src_test/include/test_bicop_sanity_checks.hpp

```diff
@@ -91,6 +91,33 @@ TEST(bicop_sanity_checks, controls_checks)
   EXPECT_ANY_THROW(controls.set_psi0(1.0));
 }
 
+TEST(bicop_sanity_checks, fit_controls_config_works)
+{
+  FitControlsBicop controls;
+  FitControlsConfig config;
+  config.family_set = controls.get_family_set();
+  config.parametric_method = controls.get_parametric_method();
+  config.nonparametric_method = controls.get_nonparametric_method();
+  config.nonparametric_mult = controls.get_nonparametric_mult();
+  config.selection_criterion = controls.get_selection_criterion();
+  config.weights = controls.get_weights();
+  config.psi0 = controls.get_psi0();
+  config.preselect_families = controls.get_preselect_families();
+  config.allow_rotations = controls.get_allow_rotations();
+  config.num_threads = controls.get_num_threads();
+  FitControlsBicop controls2(config);
+  EXPECT_EQ(controls.get_family_set(), controls2.get_family_set());
+  EXPECT_EQ(controls.get_parametric_method(), controls2.get_parametric_method());
+  EXPECT_EQ(controls.get_nonparametric_method(), controls2.get_nonparametric_method());
+  EXPECT_EQ(controls.get_nonparametric_mult(), controls2.get_nonparametric_mult());
+  EXPECT_EQ(controls.get_selection_criterion(), controls2.get_selection_criterion());
+  EXPECT_EQ(controls.get_weights(), controls2.get_weights());
+  EXPECT_EQ(controls.get_psi0(), controls2.get_psi0());
+  EXPECT_EQ(controls.get_preselect_families(), controls2.get_preselect_families());
+  EXPECT_EQ(controls.get_allow_rotations(), controls2.get_allow_rotations());
+  EXPECT_EQ(controls.get_num_threads(), controls2.get_num_threads());
+}
+
```

Comment:
Should be better to set and test against non-default values of the parameters, otherwise we might miss something just because the defaults are equal in both constructors.

Reply:
Is that alright now?